app/views/dialogue.py

```diff
@@ -466,3 +459,118 @@ def _setup_messagebox(title: str | None) -> QMessageBox:
         dialogue.setWindowTitle(DEFAULT_TITLE)
 
     return dialogue
+
+
+def show_settings_error() -> None:
+    """
+    Displays a fatal error box indicating the app was
+    unable to start due to corrupt settings. Called if unable
+    to parse the settings file.
+
+    :param settings: settings model to allow resetting settings
+    """
+    logger.info("Showing settings failure box")
+    diag = SettingsFailureDialog()
+    diag.exec_()
+
+
+class SettingsFailureDialog(QDialog):
```

Comment:
Looks generally good.

I do wonder if we should look at making a base class for these dialogs and use inheritance to set custom dialogs up. Don't need to do it now in this PR.

Reply:
Agree, was thinking that too when making these revisions. May take it up in a next PR